spec/system/admin/instrument_scheduling_tab_spec.rb

```diff
@@ -0,0 +1,106 @@
+    let!(:instrument) do
+      FactoryBot.create(:setup_instrument, pricing_mode: "Duration", facility: facility)
+    end
+
+    context "new schedule rule" do
+      before do
+        visit new_facility_instrument_schedule_rule_path(facility, instrument)
+      end
+
+      it "shows common Schedule rule fields" do
+        expect(page).to have_content(instrument.name)
+        expect(page).to have_content("Add Schedule Rule")
+        expect(page).not_to have_content("Price Group Discounts")
+
+        check "Sun"
+
+        select "7", from: "schedule_rule_start_hour"
+        select "8", from: "schedule_rule_end_hour"
+
+        click_button "Create"
+
+        expect(page).to have_content("Days of Week")
+        expect(page).to have_content("Start Time")
+        expect(page).to have_content("End Time")
+        expect(page).not_to have_content("Discount (%)")
+      end
+    end
+
+    context "edit schedule rules" do
+      before do
+        visit edit_facility_instrument_schedule_rule_path(facility, instrument, instrument.schedule_rules.first)
+      end
+
+      it "shows common Schedule rule fields" do
+        expect(page).to have_content("Editing Schedule Rule")
+        expect(page).not_to have_content("Price Group Discounts")
+
+        check "Mon"
+
+        click_button "Update"
+
+        expect(page).to have_content("Days of Week")
+        expect(page).to have_content("Start Time")
+        expect(page).to have_content("End Time")
+        expect(page).not_to have_content("Discount (%)")
+      end
+    end
+  end
+
+  context "when the instrument has schedule pricing mode" do
+    let!(:instrument) do
+      FactoryBot.create(:setup_instrument, pricing_mode: "Schedule Rule", facility: facility)
+    end
+
+    context "new schedule rule" do
+      before do
+        visit new_facility_instrument_schedule_rule_path(facility, instrument)
+      end
+
+      it "shows common Schedule rule fields" do
+        expect(page).to have_content(instrument.name)
+        expect(page).to have_content("Add Schedule Rule")
+        expect(page).to have_content("Price Group Discounts")
+
+        check "Sun"
+
+        select "7", from: "schedule_rule_start_hour"
+        select "8", from: "schedule_rule_end_hour"
+
+        click_button "Create"
+
+        expect(page).to have_content("Days of Week")
+        expect(page).to have_content("Start Time")
+        expect(page).to have_content("End Time")
+        expect(page).to have_content("Discount (%)")
+      end
+    end
+
+    context "edit schedule rules" do
+      before do
+        visit edit_facility_instrument_schedule_rule_path(facility, instrument, instrument.schedule_rules.first)
+      end
+
+      it "shows all Schedule rule fields" do
+        expect(page).to have_content("Editing Schedule Rule")
+        expect(page).to have_content("Price Group Discounts")
+        expect(page).not_to have_content("Highlighted Price Groups")
```

Comment:
Can you add a code comment here about the intent of asserting that "Highlighted Price Groups" is absent?